tutorial_devops/consuming_conan_center.rst

```diff
@@ -0,0 +1,52 @@
+
+- Create a fork of the ConanCenter Github repository: https://github.com/conan-io/conan-center-index
+- Create a list of the packages and versions you need for your projects. This list can be added to the fork too, and maintained there (packages can be added and removed with PRs when the teams need them).
+- Create a script that first ``conan export`` all the packages in your list, then ``conan create --build=missing`` them. Do not add ``user/channel`` to these packages, it is way simpler to use them as ``zlib/1.2.13`` without user-channel.
+- Upload your build packages to your own server, that you use in production, instead of ConanCenter.
+
+
+This flow is relatively straightforward, and has many advantages that mitigate the above risks:
+
+- No central repository outage can affect your builds.
+- No changes in the central repository can break your projects.
+- You can customize, adapt, fix and perfectly control what versions are used, and release fixes in minutes, not weeks. You can apply customizations that wouldn't be accepted in the central repository.
+- You fully own the supply chain, from the source (recipes) to the binaries, eliminating in practice the majority of potential supply chain attacks of central repositories.
+
+
+Updating from the upstream ``conan-center-index`` Github repo is still possible, and it can be done in a fully controlled way, highly reducing risks:
+
+- Merge the latest changes in the upstream main fork of ``conan-center-index`` into your fork.
+- You can check and audit those changes if you want to, analyzing the diffs (some automation that trims the diffs of recipes that you don't use could be useful)
+- Firing the above process will efficiently rebuild the new binaries that are needed. If your recipes are not affected by changes, the process will avoid rebuilding binaries (thanks to ``--build=missing``).
+- You can upload the packages to a secondary "test" server repository. Then test your project against that test server, to check that your project is not broken by the new ConanCenter packages.
+- Once you verify that everything is good with the new packages, you can copy them from the secondary "test" repository to your main production repository to start using them.
+
+
+.. note::
+
+  **Best practices**
+
+  - Do not use ConanCenter packages on production systems, store the packages you need in your own server and use them from there.
+  - Create your own binaries from your fork to completely own the pipeline, remove all breakage risks, accelerate fixes and remove security supply chain attacks. The process is not complicated and really worth it.
```

Comment:
"remove security supply chain attacks" -> provide additional security to reduce the chance of supply chain attacks,. I don't think this is removed unless you stop relying on external source code altogether, or if you review ALL external code.